private-templates-service/client/src/components/Dashboard/PreviewModal/PreviewModal.tsx

```diff
@@ -127,7 +130,7 @@ class PreviewModal extends React.Component<Props, State> {
         {template && !isFetching ?
           <React.Fragment>
             <ACOuterPanel>
-              <StyledDropdown selectedKey={selectedItem.key} onChange={this.hostConfigChange} options={DropdownOptions} />
+              <StyledDropdown selectedKey={selectedItem.key} onChange={this.hostConfigChange} options={DropdownOptions} tabIndex={this.props.modalState ? -1 : 0} />
```

Comment:
Can I ask - why do we need tabIndex here? This dropdown only appears when the preview is open and there's no other case when it's visible